### DISCUSSION TITLE
建议官方出一个回复评论的模块

### DISCUSSION THREAD

**Discussion by CyccyYan** [share121/inter-knot#3059] [其他]
![77D1B18365D064927170AAE887C6E04B_0](https://github.com/user-attachments/assets/212a6399-8c04-4859-b369-f985b82f1a78)
有时候看见想吐槽的评论只能发评论区

**Comment by yzysrewrwqr** [comment]
如果能不跳转就更好了，直接整个论坛<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Reply by CyccyYan** [reply]
同意，每次发评论都跳转的话，对于网络不佳的人可能有些太折磨了
![2f8b17407e8e98424162198610e2499f](https://github.com/user-attachments/assets/c1e70400-e44d-40df-a831-4ca933b07cd5)

**Reply by rong-xiaoli** [reply]
不跳转的话貌似得要每个人的GitHub API token才可以工作，这样的话上手难度较高。不过也行，可以这么做。

**Comment by LT-UT** [comment]
发到评论区的话就每一个评论过的人就都会受到邮件

**Comment by CyccyYan** [comment]
原来这玩意可以恢复的吗！

**Reply by CyccyYan** [reply]
回复

**Reply by LT-UT** [reply]
我这又收到了

**Reply by CyccyYan** [reply]
我也收到了<img src="https://bbs-static.miyoushe.com/static/2023/11/16/a6976b54880a512092a19f34c1cd3931_2328184749682335286.png" alt="铃-萌混过关" width="75" >

**Reply by Pheathon** [reply]
可以哦
